packages/admin/admin/src/alerts/AdminComponentAlerts.tsx

```diff
@@ -0,0 +1,156 @@
+import { ArrowRight, Check, Close, Error, Info, Warning } from "@comet/admin-icons";
+import { Box, Button, Container, Theme, Typography } from "@mui/material";
+import { createStyles, WithStyles, withStyles } from "@mui/styles";
+import clsx from "clsx";
+import * as React from "react";
+import { FormattedMessage } from "react-intl";
+
+export interface AdminComponentAlertsProps {
+    severity?: "info" | "warning" | "error" | "success";
+    startIcon?: React.ReactNode;
+    title?: React.ReactNode;
+    text: React.ReactNode;
+    hideCloseButton?: boolean;
+    onCloseClick?: () => void;
+}
+
+export type AdminComponentAlertsClassKey =
+    | "root"
+    | "content"
+    | "severityInfo"
+    | "severityWarning"
+    | "severityError"
+    | "severitySuccess"
+    | "title"
+    | "text"
+    | "button"
+    | "container"
+    | "closeIconFull"
+    | "marginBottom"
+    | "paddingLeft"
+    | "cursorPointer";
+
+const styles = (theme: Theme) =>
+    createStyles<AdminComponentAlertsClassKey, AdminComponentAlertsProps>({
+        root: {
+            backgroundColor: theme.palette.background.paper,
+            borderRadius: 4,
+            width: 400,
+            padding: 20,
+            display: "flex",
+            boxShadow: "0px 0px 20px 0px rgba(0, 0, 0, 0.10)",
+            position: "relative",
+        },
+        content: {},
+        title: {
+            fontWeight: "bold",
+        },
+        text: {
+            fontWeight: 300,
+            fontSize: 14,
+            whiteSpace: "nowrap",
+        },
+        button: {
+            paddingLeft: 0,
+        },
+        container: {
+            display: "flex",
+            alignItems: "center",
+        },
+        closeIconFull: {
+            position: "absolute",
+            right: 10,
+            top: 10,
+        },
+        marginBottom: {
+            marginBottom: 10,
+        },
+        cursorPointer: {
+            cursor: "pointer",
+        },
+        paddingLeft: {
+            paddingLeft: 180,
+            paddingRight: 10,
+        },
+        severityInfo: {
+            border: `1px solid ${theme.palette.primary.main}`,
+            borderLeft: `5px solid ${theme.palette.primary.main}`,
+        },
+        severityWarning: {
+            border: `1px solid ${theme.palette.warning.main}`,
+            borderLeft: `5px solid ${theme.palette.warning.main}`,
+        },
+        severityError: {
+            border: `1px solid #D11700`,
+            borderLeft: `5px solid #D11700`,
+        },
+        severitySuccess: {
+            border: `1px solid ${theme.palette.secondary.main}`,
+            borderLeft: `5px solid ${theme.palette.secondary.main}`,
+        },
```

Comment:
@thomasdax98 We discussed theme-changes recently with UX, also regarding the `error` and `success` colors. 
Should we wait until we updated the theme so we can use those here instead of using the hardcoded value & the `secondary` that we don't really use anywhere else, or should we leave it like this for now? 
Do we have a timeline for the theme-changes?